build.sbt

```diff
@@ -1,6 +1,6 @@
 lazy val alpakka = project
   .in(file("."))
-  .enablePlugins(PublishUnidoc)
+//  .enablePlugins(PublishUnidoc)
```

Comment:
You couldn't compile with these uncommented?

Reply:
Nope, got errors related to Publish.scala so I got rid of the whole thing. Didn't seem like an integral part.